components/proposals/20220121-notebook-idleness.md

```diff
@@ -0,0 +1,163 @@
+# Extend Notebooks Controller to expose idleness for Jupyter

```

Comment:
Nit, for the name of the file: Could you rename the file to `20220121-jupyter-notebook-idleness.md`?

This will make it slightly more clear that this proposal targets Jupyter Notebooks, and will help with organization if in the future we add proposals for VSCode etc

Reply:
**DONE!** I added a "review: rename the proposal markdown file" commit to resolve this.